src/libs/actions/PaymentMethods.ts

```diff
@@ -269,21 +285,20 @@ function dismissSuccessfulTransferBalancePage() {
 
 /**
  * Looks through each payment method to see if there is an existing error
- * @param {Object} bankList
- * @param {Object} fundList
- * @returns {Boolean}
+ *
  */
-function hasPaymentMethodError(bankList, fundList) {
+function hasPaymentMethodError(bankList: OnyxValues[typeof ONYXKEYS.BANK_ACCOUNT_LIST], fundList: OnyxValues[typeof ONYXKEYS.FUND_LIST]): boolean {
     const combinedPaymentMethods = {...bankList, ...fundList};
-    return _.some(combinedPaymentMethods, (item) => !_.isEmpty(item.errors));
+
+    return Object.values(combinedPaymentMethods).some((item) => !!item.errors);
```

Comment:
This logic isn't the same `!_.isEmpty(item.errors)` vs `!!item.errors`. `errors` is of type `Record<string, string> | undefined` so in order to ensure the object isn't empty use `Object.keys(item.errors ?? {}).length`